setup.py

```diff
@@ -33,14 +33,15 @@
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
         'Programming Language :: Python :: 3.11',
+        'Programming Language :: Python :: 3.12',
```

Comment:
usually I wouldn't declare an unreleased version, but I see trio does it

Reply:
Yeah.  FWIW [CPython is up to rc3, and 3.12.0 final is due Monday](https://peps.python.org/pep-0693/#schedule), so we're not far in advance here.